shells/pipes-shell/web/deploy/android_deploy.sh

```diff
@@ -7,10 +7,7 @@ OUT_DIR="$PWD/$1"
 cd "${0%/*}"
 
 # Webpack shell code.
-npx webpack --display=errors-only --output="$OUT_DIR/shell.js"
-
-# Copy over webpacked Arcs runtime.
-cp ../../../lib/build/worker.js "$OUT_DIR/worker.js"
```

Comment:
Looks like we can also remove the build dep "//tools:sigh_webpack" at shells/pipes-shell/BUILD
since we had to run "sigh webpack" via "//tools:sigh_webpack" in the past to generate a bundled worker.js at shells/lib/build then copied it over at deploy scripts. Now that the webpack config script takes care of both shell.js and worker.js, it should be safe to remove unused dep "//tools:sigh_webpack"

Reply:
👍